hack/update-kubevirtci.sh

```diff
@@ -17,7 +17,7 @@
 SCRIPT_ROOT="$(cd "$(dirname $0)/../" && pwd -P)"
 
 # the kubevirtci commit hash to vendor from
-kubevirtci_git_hash=142f7450f94d866db863b4af254b5428fe1c570a
+kubevirtci_git_hash=91f55e31a421ae85a2f66200d81d62c80b632581
```

Comment:
lines 28 , 30 are not needed anymore
line 30 always take the dynamic latest, so its not right, because the kubevirtci is frozen via this line and
because there is 
https://github.com/kubevirt/containerized-data-importer/pull/1886/files#diff-71475b250804182f279cfa3d58a9336f4015fa588a6216f05f7cd01599ba1bedR42

(keep me honest here)

Reply:
Yep you're right, the commit hash does not play a part, can just dl using the tag

Reply:
Done